config/304-xslttransform.yaml

```diff
@@ -89,14 +90,39 @@ spec:
                 - required: [value]
                 - required: [valueFromSecret]
                 - required: [valueFromConfigMap]
-
               allowPerEventXSLT:
                 description: Whether the XSLT informed at the spec can be overriden at each CloudEvent.
                 type: boolean
             anyOf:
             - required: [xslt]
             - required: [allowPerEventXSLT]
-
+              sink:
+                description: The destination of events transformed by this component.
+                type: object
+                properties:
+                  ref:
+                    description: Reference to an addressable Kubernetes object to be used as the destination of events.
```

Comment:
Can we explain that in the absence of it, the transformed event is sent back to the caller?